Example/example.ts

```diff
@@ -1,13 +1,20 @@
 import { Boom } from '@hapi/boom'
+import parsePhoneNumber from 'libphonenumber-js'
 import NodeCache from 'node-cache'
-import makeWASocket, { AnyMessageContent, delay, DisconnectReason, fetchLatestBaileysVersion, getAggregateVotesInPollMessage, makeCacheableSignalKeyStore, makeInMemoryStore, proto, useMultiFileAuthState, WAMessageContent, WAMessageKey } from '../src'
-import MAIN_LOGGER from '../src/Utils/logger'
+import P from 'pino'
+import readline from 'readline'
+import makeWASocket, { AnyMessageContent, delay, DisconnectReason, fetchLatestBaileysVersion, getAggregateVotesInPollMessage, makeCacheableSignalKeyStore, makeInMemoryStore, PHONENUMBER_MCC, proto, useMultiFileAuthState, WAMessageContent, WAMessageKey } from '../src'
 
-const logger = MAIN_LOGGER.child({ })
-logger.level = 'trace'
+const logger = P({
```

Comment:
(Subjective opinion part)
 Personally I prefer the "not so pretty but compact" logging as we have it on the master branch right now. The advantages of the prettier are obvious, but the "non-pretty" - for me - is more convenient because it only produces one line for each log statement so in an editor without line break it's easier to skim through logs. But it's only the example so it may not matter as much

(Objective opinion part)
I can't tell why but on this PR the `pair-device` log comes after the QR code. This way it nudges away the QR code if you don't enlarge your terminal window. On the master it's the bottom-most part in the terminal (until new messages come in of course). Do you happen to know why that is?

Reply:
I've reverted the logger changes